src/status_im/ui/screens/discover/components/views.cljs

```diff
@@ -0,0 +1,82 @@
+(ns status-im.ui.screens.discover.components.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :refer [subscribe dispatch]]
+            [clojure.string :as str]
+            [status-im.components.react :refer [view text image touchable-highlight]]
```

Comment:
We want to avoid using refers as much as possible, I think most if not at all of these can be qualified namespaces. `react/`, `identicon/` etc.

Reply:
Agree, refactored it but stopped half way, see reply to one of the other comments.

Reply:
Ah right, it's copy-paste from other file? Thought it was a new one. Yeah sure, makes sense